pandas/tools/tests/test_util.py

```diff
@@ -401,6 +401,32 @@ def test_downcast(self):
             res = pd.to_numeric(data, downcast=downcast)
             tm.assert_numpy_array_equal(res, expected)
 
+        # check that the smallest and largest values in each integer type pass to each type.
+        int8 = [-128, 127]
```

Comment:
can you do this in a loop instead (eg. a loop of tuples of the start,stop, dtype)